frontend/assets/koblenz/l10n/override_de.json

```diff
@@ -1,51 +1,55 @@
-    "applyDescription": "Sie haben noch keinen KoblenzPass? Hier können Sie Ihren KoblenzPass beantragen.",
-    "cardExpired": "Ihr Pass ist abgelaufen. Unter \"Weitere Aktionen\" können Sie Ihren Pass verlängern oder einen neuen Pass beantragen.",
-    "cardInvalid": "Ihr Pass ist ungültig. Er wurde entweder widerrufen oder auf einem anderen Gerät aktiviert.",
-    "cardNotYetValid": "Der Gültigkeitszeitraum Ihres Passes hat noch nicht begonnen.",
-    "checkFailed": "Ihr Pass konnte nicht auf seine Gültigkeit geprüft werden. Bitte stellen Sie sicher, dass eine Internetverbindung besteht und prüfen Sie erneut.",
-    "codeRevoked": "Dieser Pass konnte nicht aktiviert werden, da er widerrufen wurde.",
-    "extendCard": "Pass verlängern",
-    "extendCardNotificationTitle": "Hinweis zum Ablauf des Passes",
-    "extendCardNotificationDescription": "Ihr Pass läuft demnächst ab.\nVerlängern Sie ihn jetzt, um Ihre Vorteile weiterhin nutzen zu können.",
-    "moreActionsActivateDescription": "Ihr hinterlegter KoblenzPass bleibt erhalten. Sie können diesen manuell entfernen.",
+    "activateCurrentDeviceDescription": "Ihr KoblenzPass wurde bereits auf einem anderen Gerät aktiviert. Wenn Sie fortfahren, wird Ihr KoblenzPass automatisch auf dem anderen Gerät deaktiviert, falls er dort noch aktiv ist.",
+    "activateCurrentDeviceTitle": "KoblenzPass auf diesem Gerät hinzufügen?",
+    "activateDescription": "Sie haben Ihren KoblenzPass bereits aktiviert und ein Dokument mit einem Aktivierungscode (QR-Code) oder den KoblenzPass als Papierausdruck erhalten? Scannen Sie den QR-Code hier ein.",
+    "activateTitle": "Hinzufügen",
+    "applyTitle": "Aktivieren",
+    "applyDescription": "Sie haben noch keinen KoblenzPass? Hier können Sie den KoblenzPass mit Ihrem Aktenzeichen aktivieren.",
+    "cardExpired": "Ihr KoblenzPass ist abgelaufen. Unter \"Weitere Aktionen\" können Sie Ihren KoblenzPass verlängern oder einen neuen KoblenzPass beantragen.",
+    "cardInvalid": "Ihr KoblenzPass ist ungültig. Er wurde entweder gesperrt oder auf einem anderen Gerät aktiviert.",
+    "cardNotYetValid": "Der Gültigkeitszeitraum Ihres KoblenzPasses hat noch nicht begonnen.",
+    "checkFailed": "Ihr KoblenzPass konnte nicht auf seine Gültigkeit geprüft werden. Bitte stellen Sie sicher, dass eine Internetverbindung besteht und versuchen Sie es erneut.",
+    "codeRevoked": "Dieser KoblenzPass konnte nicht aktiviert werden, da er gesperrt wurde.",
+    "extendCard": "Verlängern",
+    "extendCardNotificationTitle": "Hinweis zum Auslaufen des KoblenzPasses",
+    "extendCardNotificationDescription": "Ihr KoblenzPass läuft demnächst ab. Verlängern Sie jetzt Ihren KoblenzPass, um die Vorteile weiterhin nutzen zu können.",
+    "moreActionsActivateDescription": "Sie haben einen weiteren KoblenzPass bereits aktiviert und ein Dokument mit einem Aktivierungscode (QR-Code) erhalten (z. B. für Familienangehörige)? Scannen Sie den QR-Code hier ein.",
```

Comment:
```suggestion
    "moreActionsActivateDescription": "Sie haben bereits einen weiteren KoblenzPass aktiviert und ein Dokument mit einem Aktivierungscode (QR-Code) erhalten (z. B. für Familienangehörige)? Scannen Sie den QR-Code hier ein.",
```
or even
```suggestion
    "moreActionsActivateDescription": "Sie haben bereits einen KoblenzPass aktiviert und ein Dokument mit einem Aktivierungscode (QR-Code) erhalten (z. B. für Familienangehörige)? Scannen Sie den QR-Code hier ein.",
```

Reply:
As Koblenz has a different wording regarding application and activation I'd prefer your first suggestion. It feels like the second one is not that clear that it targets a nth+1 pass.